README.md

```diff
@@ -302,33 +305,34 @@ Here is just a quick taste of what RocketPy is able to calculate. There are hund
 
 # Authors and Contributors
 
-This package was originally created by [Giovani Ceotto](https://github.com/giovaniceotto/) as part of his work at [Projeto Jupiter](https://github.com/Projeto-Jupiter/). [Rodrigo Schmitt](https://github.com/rodrigo-schmitt/) was one of the first contributors.
+This package was originally created by [Giovani Ceotto](https://github.com/RocketPy-Team/) as part of his work at [Projeto Jupiter](https://github.com/RocketPy-Team/). [Rodrigo Schmitt](https://github.com/rodrigo-schmitt/) was one of the first contributors.
```

Comment:
This does not seem right.

Reply:
I'm sorry, I've used a simple "replace all" and did not expect these behavior. I fixed now.